opendebates/context_processors.py

```diff
@@ -63,6 +63,7 @@ def _get_categories():
         'ALLOW_SORTING_BY_VOTES': mode.allow_sorting_by_votes,
         'ALLOW_VOTING_AND_SUBMITTING_QUESTIONS': mode.allow_voting_and_submitting_questions,
         'DEBATE_TIME': mode.debate_time,
+        'PREVIOUS_DEBATE_TIME': mode.previous_debate_time,
```

Comment:
Per my comments elsewhere I think we may actually want this to be more like `mode.previous_debate_time if mode.previous_debate_time and mode.previous_debate_time < now() else None`.